recipes/default.rb

```diff
@@ -4,3 +4,7 @@
 include_recipe 'midokura::midolman'
 include_recipe 'midokura::midonet-api'
 #include_recipe 'midokura::midonet-cp'
+
+execute 'install openssl-devel' do
```

Comment:
I think we will need to move this above the include_recipe cassandra line since cassandra is the one pulling in thrift.